src/NDS.cpp

```diff
@@ -177,6 +177,9 @@ void SetGBASlotTimings();
 
 bool Init()
 {
+    NumFrames = 0;
```

Comment:
shouldn't this happen on reset?

Reply:
This is so that time doesn't reset by just resetting (which a user will have to reset if they change the time in firmware).

Reply:
we call Init only once when the program is started

Reply:
Which is exactly the intent, to not reset frame counters unless the entire program is reset.